src/main/java/org/irods/Serialize/ModifyPermissionsOperations.java

```diff
@@ -18,18 +18,18 @@ public class ModifyPermissionsOperations {
     @JsonProperty("acl")
     private String acl;
 
-    public ModifyPermissionsOperations(String entity_name, Permission acl) throws IrodsException {
+    public ModifyPermissionsOperations(String entity_name, Permission perm) {
```

Comment:
Need to include the `throws` specifier so the dev knows the constructor can throw exceptions.

Reply:
Done.